internal/helm/controller/controller.go

```diff
@@ -93,6 +108,21 @@ func Add(mgr manager.Manager, options WatchOptions) error {
 	return nil
 }
 
+// parsePredicateSelector parses the selector in the WatchOptions and creates a predicate
+// that is used to filter resources based on the specified selector
+func parsePredicateSelector(selector metav1.LabelSelector) (ctrlpredicate.Predicate, error) {
+	// If a selector has been specified in watches.yaml, add it to the watch's predicates.
+	if !reflect.ValueOf(selector).IsZero() {
+		p, err := ctrlpredicate.LabelSelectorPredicate(selector)
+		if err != nil {
+			return nil, fmt.Errorf("Error constructing predicate from watches selector: %v", err)
```

Comment:
Error messages must start with a lowercase character if alphabetical.
```suggestion
			return nil, fmt.Errorf("error constructing predicate from watches selector: %v", err)
```